VisualPinball.Unity/VisualPinball.Unity.Editor/Managers/Lamp/LampListViewItemRenderer.cs

```diff
@@ -136,8 +136,6 @@ private void RenderCoilId(Dictionary<string, LampState> lampStatuses, LampListDa
 			}
 			cellRect.x += 20;
 			cellRect.width -= 20;
-
-			EditorGUI.LabelField(cellRect, lampListData.InternalId.ToString());

```

Comment:
I think you can remove the cellRect part as well.

Reply:
Done.